src/include/duckdb/function/udf_function.hpp

```diff
@@ -0,0 +1,303 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/function/udf_function2.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/common/types.hpp"
+#include "duckdb/function/scalar_function.hpp"
+#include "duckdb/main/client_context.hpp"
```

Comment:
Could you remove this include? We want to avoid including client_context in central places (connection.hpp) to avoid small changes triggering massive re-compiles.